kinto_megaphone/heartbeat.py

```diff
@@ -0,0 +1,25 @@
+from pyramid.config import ConfigurationError
+from pyramid.settings import aslist
+
+
+class MegaphoneHeartbeat(object):
+    def __init__(self, client):
+        self.client = client
+
+    def __call__(self, _request):
+        data = self.client.heartbeat()
+        return data['database'] == 'ok' and data['status'] == 'ok'
+
+
+def find_megaphone_url(registry):
+    # FIXME: this assumes only one Megaphone server per config
+    listeners = aslist(registry.settings['event_listeners'])
+    for listener in listeners:
+        prefix = 'event_listeners.{}'.format(listener)
+        listener_modpath = registry.settings['{}.use'.format(prefix)]
```

Comment:
This could raise `KeyError`. Use `.get(..., "")`

Reply:
I assumed this was a configuration error that would be caught for me by kinto initialization code. Looking at it deeper, it does seem like `kinto.includes` gets scanned after setting up listeners, which is where I would expect a ConfigurationError to get raised. On the other hand, looking at the code, it seems like `.use` isn't really *required* -- there's another code path that I don't really understand. Maybe I can put a whole module path instead of just an alias? This isn't covered by the documentation.

Anyhow, I don't mind raising a `ConfigurationError` here just to be safe, unless it's not actually an error, in which case maybe I need to do something different here?

Reply:
If I understand this correctly : 

https://github.com/Kinto/kinto/blob/7a29b180bc0ba3872b2360ed7f52046ecc8aa7fa/kinto/core/initialization.py#L431-L441

If the listener that you put is not an alias (ie. is a module path) and has a `load_from_config()` function then `.use` is not required. 

We can deprecate one of them. I'd prefer the config with `.use` btw